app/models/sir_trevor_rails/blocks/browse_block.rb

```diff
@@ -32,7 +32,7 @@ def display_item_counts?
       def as_json
         result = super
 
-        result[:data][:item].each do |_k, v|
+        result[:data][:item].to_h.each_value do |v|
```

Comment:
I wonder if we should revisit how this blocks get serialized to json. Maybe we're missing a useful class or other way to inject custom serializations (rather than munging it after it's been serialized)